client/src/reducers/socket.js

```diff
@@ -9,13 +9,18 @@ export default function SocketReducer(state = {
 			message: action.message
 		};
 	case 'SOCKET_NEW_TICKET': {
-		const newTicket = action.ticket;
-		newTicket.timestamp = new Date(newTicket.timestamp);
+		const { timeFiled } = action.newTicket;
+		const ticketWithDate = { ...action.newTicket, timeFiled: new Date(timeFiled) };
 		return {
 			...state,
-			tickets: [...state.tickets, newTicket]
+			tickets: [...state.tickets, ticketWithDate]
 		};
 	}
+	case 'SOCKET_ERROR': {
+		const error = { ...action.error };
```

Comment:
Why `...`? 
And I think you can return some state with a new field called errorMsg so we can eventually show an error msg on the webpage.